app/services/csv_exporter.rb

```diff
@@ -5,8 +5,10 @@ def self.export(resources, columns)
     Enumerator.new do |csv|
       csv << csv_header(columns)
 
-      resources.reorder(nil).limit(nil).find_each do |obj|
-        csv << CSV.generate_line(columns.map{|c| obj.send(c)})
+      resources.uncached do
```

Comment:
uncached is needed to prevent memory leaking when processing a huge amount of records - this disables `ActiveRecord::QueryCache` for the loop, preventing rails from trying to cache potentially millions of rows.